File: server/src/main/kotlin/com/evgenltd/financemanager/importdata/entity/ImportDataEntry.kt
package com.evgenltd.financemanager.importdata.entity

import com.evgenltd.financemanager.common.util.Amount
import com.evgenltd.financemanager.transaction.entity.Direction
import java.time.LocalDate

class ImportDataEntry(
        var id: String,
        var date: LocalDate,
        var direction: Direction,
        var amount: Amount,
        var description: String,
        var imported: Boolean
)

File: web/src/app/import-data/model/import-data-entry.ts
import {Amount} from "../../common/model/amount";

export class ImportDataEntry {
  id!: string
  fromDb!: boolean
  fromFile!: Boolean
  systemId!: String
  date!: string
  direction!: 'IN' | 'OUT'
  amount!: Amount
  description!: string
  selected: boolean = false
}
